src/bionics.cpp

```diff
@@ -887,42 +888,29 @@ bool player::uninstall_bionic( bionic_id const &b_id, int skill_level, bool auto
     }
 
     // removal of bionics adds +2 difficulty over installation
-    int chance_of_success;
-    if( skill_level != -1 ) { // NPC surgeon removal
```

Comment:
Why did you remove this case? NPC surgeon isn't using autodoc to remove bionics, so he doesn't need to configure it and therefore doesn't need to check for computers skill.

Reply:
I was hoping to get rid of the special case: there's always an installer who may or may not be the PC. If some installers should have extra skill in some instances, I'd rather flag it in the skill calculation.

Doing it my way also removes an extra copy of the code to calculate skill in the failure path.

Reply:
Went back and added the skill override, then refactored it substantially so the adjusted skill is returned in a separate function and carried around for the failure case.